pkg/registry/resource/rest/storage_resource.go

```diff
@@ -50,6 +49,10 @@ func (p RESTStorageProvider) NewRESTStorage(apiResourceConfigSource serverstorag
 func (p RESTStorageProvider) v1alpha3Storage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) (map[string]rest.Storage, error) {
 	storage := map[string]rest.Storage{}
 
+	// Registered also without the corresponding feature gate because it
```

Comment:
I've used "The REST storage registers resource kinds ..." and moved it to the top of the file because it's a general approach that will also apply to the upcoming `v1beta1Storage`.

I actually have a follow-up commit where I change this because on Slack it was suggested that feature gates *should* influence type registration. But let's not discuss that as part of this PR and just make sure that this note is preserved.